rl/wolf_sheep/train_config.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.policy.policy import PolicySpec
+
+from .model import WolfSheepRL
+
+
+# Configuration to train the model
+# Feel free to adjust the configuration as necessary
+def env_creator(_):
+    return WolfSheepRL(
+        width=20,
+        height=20,
+        initial_sheep=100,
+        initial_wolves=25,
+        sheep_reproduce=0.04,
+        wolf_reproduce=0.05,
+        wolf_gain_from_food=20,
+        grass=True,
+        grass_regrowth_time=30,
+        sheep_gain_from_food=4,
+    )
+
+
+config = {
+    "env_name": "WorldSheepModel-v0",
+    "env_creator": env_creator,
+    "framework": "torch",  # Assuming you want to use PyTorch
+    "train_batch_size": 150,  # Assuming a default value, adjust as necessary
+    "policies": {
+        "policy_sheep": PolicySpec(config=PPOConfig.overrides(framework_str="torch")),
+        "policy_wolf": PolicySpec(config=PPOConfig.overrides(framework_str="torch")),
+    },
+    "policy_mapping_fn": lambda agent_id, *args, **kwargs: "policy_sheep"
+    if agent_id[0:5] == "sheep"
```

Comment:
I am genuinely curious here, why is this only 0 to 5?

Reply:
Ah, this.  Basically, agent_id is "Sheep[number]", and wolf is "Wolf[number]". So,  we are checking the first 5 letters in Sheep to partition it. I will add a comment